config.yaml

```diff
@@ -76,6 +76,18 @@ options:
     type: string
     default: ""
     description: The username to use to access swift.
+  tls-ca-blob:
+    type: string
+    default: ""
+    description: Base64 encoded TLS CA certificate (overwrites tls-cert-path file).
+  tls-cert-blob:
+    type: string
+    default: ""
+    description: Base64 encoded TLS certificate (overwrites tls-cert-path file).
+  tls-key-blob:
+    type: string
+    default: ""
+    description: Base64 encoded TLS certificate private key (overwrites tls-key-path file).
```

Comment:
are there doc updates showing how to use this? Should there be?

Reply:
I could add them to README, but I thought we were moving config docs out of README and into the generated Juju charm store?

Reply:
I was thinking more https://www.ubuntu.com/kubernetes/docs/docker-registry. A blurb with how to use on there.

Reply:
Ah good point

Reply:
https://github.com/juju-solutions/kubernetes-docs/pull/61/files